### PR TITLE
Remove service install

### PR DESCRIPTION
**Pull request by CGNonofr**
based on https://github.com/CodinGame/monaco-vscode-api/pull/96

Just simplify the api by removing remaining stuff (by pluging it on vscode services)

### CODE REVIEW

**Review by kaisalmen — APPROVED** [review]
LGTM. Wonderful work on all four PRs. 🎉

File: src/polyfill.ts [hunk 1]
```diff
@@ -16,28 +16,31 @@ import { LanguageService as VScodeLanguageService } from 'vscode/vs/editor/commo
...
+import { WorkspaceFolder as MonacoWorkspaceFolder, Workspace as MonacoWorkspace } from 'monaco-editor/esm/vs/platform/workspace/common/workspace.js'
+import { IWorkspaceContextService, WorkbenchState, WorkspaceFolder as VScodeWorkspaceFolder, Workspace as VScodeWorkspace } from 'vscode/vs/platform/workspace/common/workspace.js'
 import { List as MonacoList } from 'monaco-editor/esm/vs/base/browser/ui/list/listWidget.js'
 import { List as VScodeList } from 'vscode/vs/base/browser/ui/list/listWidget.js'
 import { Color as MonacoColor, RGBA as MonacoRGBA } from 'monaco-editor/esm/vs/base/common/color.js'
 import { Color as VScodeColor, RGBA as VScodeRGBA } from 'vscode/vs/base/common/color.js'
 import { LogService as MonacoLogService } from 'monaco-editor/esm/vs/platform/log/common/logService.js'
 import { LogService as VScodeLogService } from 'vscode/vs/platform/log/common/logService.js'
+import { ConsoleLogger as MonacoConsoleLogger, MultiplexLogger as MonacoMultiplexLogger } from 'monaco-editor/esm/vs/platform/log/common/log'
+import { ConsoleLogger as VScodeConsoleLogger, MultiplexLogger as VScodeMultiplexLogger } from 'vscode/vs/platform/log/common/log'
```
Comment (kaisalmen) [inline src/polyfill.ts:41]: This must be log.js

Comment (kaisalmen) [reply src/polyfill.ts:41]: It breaks webpack build again

**Review by kaisalmen — CHANGES_REQUESTED** [review]
See comment about log.js. It breaks the webpack build in monaco-languageclient again.

File: src/polyfill.ts (hunk 1 above)
Comment (CGNonofr) [reply src/polyfill.ts:41]: Oops! here ce21dc83293d330318ce2ff64b577dbf8d843f92 to make sure it doesn't happen again

**Review by kaisalmen — APPROVED** [review]
Checked the latest delta again it's looking good. LGTM

**Pull request merged**